Xamarin.Forms.Xaml.UnitTests/Issues/Bz53561.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Yea I'll fix up the unit test a bit.  I couldn't get that project to compile.  There seemed to be alot of issues with it, maybe because I used vs2017?  I'll try my best to get it right today, is the project supposed to compile?